Expecto/Expecto.fs

```diff
@@ -387,9 +384,7 @@ module Tests =
       "--filter-test-list", "Filters the list of test lists by a given substring.", Args.string Filter_Test_List
       "--filter-test-case", "Filters the list of test cases by a given substring.", Args.string Filter_Test_Case
       "--run", "Runs only provided list of tests.", Args.list Args.string Run
-      "--list-tests", "Don't run tests, but prints out list of tests instead.", Args.none List_Tests
-      "--list-focused-tests", "Don't run tests, but prints out list of focused tests instead.", Args.none List_Focused_Tests
-      "--list-pending-tests", "Don't run tests, but prints out list of pending tests instead.", Args.none List_Pending_Tests
+      "--list-tests", "Don't run tests, but prints out list of tests instead. Lists only tests with specified state(s), or all tests if not.", Args.list (Args.focusState) List_Tests
```

Comment:
Perhaps we should make it a `--summary-state` or something similar to the existing "display only" mnemonics?

Reply:
you mean let `--list-tests` alone but add `--summary-state` to display states?

Reply:
what you think about just add states (or/and maybe colors) to `--list-tests`?

```
P pending_test.case.one 
F focused_test.case.two
N normal_test.case.three 
```

Reply:
I didn't realise we had this flag from before; let's leave it alone; if it can work without an argument (as a default showing everything) that's good. Also good would be your latest suggestion about displaying state next to each test.

Reply:
Sorry for delay, I have added a "state char" before test names. Updated PR description. 👎 or 👍 ?